src/index.js

```diff
@@ -325,46 +334,57 @@ async function handleSendTransaction(engine, payload, end) {
 				return end(`Not able to get user account`);
 			}
 			console.info(`User account fetched`);
-			let message = engine.messageToSign;
-			let nonce = await _getUserContractNonce(account, engine);
-			if(!nonce) {
-				let error = formatMessage(RESPONSE_CODES.USER_ACCOUNT_NOT_FOUND ,`User is not a registered biconomy user`);
-				eventEmitter.emit(EVENTS.BICONOMY_ERROR, error);
-				end(error);
+			if(api.url=="/native"){
```

Comment:
Use a constant here for comparison as "/native" string is used twice in the code.